feeds.go

```diff
@@ -101,3 +96,59 @@ type FeedTag struct {
 	Story               StoryMedia `json:"story"`
 	Status              string     `json:"status"`
 }
+
+func (ft *FeedTag) setValues() {
+	for i := range ft.RankedItems {
+		ft.RankedItems[i].media = &FeedMedia{
+			inst:   ft.inst,
+			NextID: ft.RankedItems[i].ID,
+		}
+	}
+
+	for i := range ft.Images {
+		ft.Images[i].media = &FeedMedia{
+			inst:   ft.inst,
+			NextID: ft.Images[i].ID,
+		}
+	}
+}
+
+// Next paginates over hashtag feed.
+func (ft *FeedTag) Next() bool {
+	if ft.err != nil {
+		return false
+	}
+
+	insta := ft.inst
+	name := ft.Name
+	body, err := insta.sendRequest(
+		&reqOptions{
+			Query: map[string]string{
+				"max_id":     ft.NextID,
+				"rank_token": insta.rankToken,
+			},
+			Endpoint: fmt.Sprintf(urlFeedTag, name),
+		},
+	)
+	if err == nil {
+		newFT := FeedTag{}
+		err = json.Unmarshal(body, &newFT)
+		if err == nil {
+			*ft = newFT
```

Comment:
Are these lines the same as yours?
```go
		newFT := &FeedTag{}
		err = json.Unmarshal(body, newFT)
		if err == nil {
			ft = newFT
```

Reply:
Yes it's the same, i copy the code from hashtags.go

Reply:
Alright, Later I will change both of them.